docs/Makefile

```diff
@@ -20,3 +20,8 @@ docs-up-docker:
 		-p 4000:4000 \
 		-v $(PWD):/srv/jekyll \
 		jekyll/jekyll:3.8 jekyll serve --livereload
+
+docs-images-resize:
+	docker run --rm -it \
+ 		-v $(PWD)/$(PATH_TO_IMAGES):/imgs \
+ 		dpokidov/imagemagick /imgs/*.$(EXTENTION) -set filename:f "%t" -resize 800x600 /imgs/%[filename:f].$(EXTENTION)
```

Comment:
how do I set the correct PATH_TO_IMAGES and EXTENTION (pls note a typo in this word) variables correctly?

Why the resolution is 800x600? Does it respect aspect ratio? 

Should we try converting to webp instead?